db/migrate/20181114094606_add_last_seen_at_to_all_tables.rb

```diff
@@ -0,0 +1,36 @@
+class AddLastSeenAtToAllTables < ActiveRecord::Migration[5.1]
+  def change
+    add_column :container_groups, :last_seen_at, :datetime
+    add_index :container_groups, :last_seen_at
```

Comment:
Pretty sure you can just pass `:index => true` to `add_column`

Reply:
ah, looks like you can't do that :-)

Reply:
Hm okay, we do it here https://github.com/ManageIQ/manageiq-schema/blob/master/db/migrate/20180920085721_add_maintenance_zone_id_to_region.rb#L3
:worried:

Reply:
ah, right, yeah it's funky, it doesn't throw exception, but it doesn't add the index either :-)